src/tests/ftest/util/security_test_base.py

```diff
@@ -25,9 +25,32 @@
 import os
 import random
 from general_utils import pcmd
+from apricot import TestWithServers
+from general_utils import create_local_file
+
+
+class SecurityTestBase(TestWithServers):
+    """Base security test class.
+
+    :avocado: recursive
+    """
+
+    def __init__(self):
```

Comment:
(pylint-super-init-not-called) __init__ method from base class 'TestWithServers' is not called